### ISSUE TITLE
Solving ODE system

### ISSUE THREAD

**Issue by Ishrak-DataScience** [lululxvi/deepxde#70]
This might be a silly question. I am trying to solve ode system d2y/d2x=6x and solution y=x^3 and dy/dx=2x with solution y=x^2. But I am unable to solve it.
![issue2](https://user-images.githubusercontent.com/42507424/85507826-f2b5b680-b62d-11ea-8124-72125bc4d672.PNG)
![issue2plot](https://user-images.githubusercontent.com/42507424/85507878-ffd2a580-b62d-11ea-8f06-257d0f581701.PNG)
Thanks in advance.

**Comment by lululxvi** [comment]
It seems that the solution is almost correct. What is the loss?

**Comment by Ishrak-DataScience** [comment]
L2 relative error: 3.5243413. I am not sure because of the green dots(prediction). They should align with the blue cross.

**Comment by lululxvi** [comment]
What is the training loss versus iterations?

**Comment by Ishrak-DataScience** [comment]
![Training loss vs epoch](https://user-images.githubusercontent.com/42507424/85813017-1ba88980-b79d-11ea-975f-6b8b90f49dcd.PNG)

**Comment by lululxvi** [comment]
In line 31, why `bc2r` is duplicated? also, use `num_boundary=2`, because there is only two boundary points.

**Comment by Ishrak-DataScience** [comment]
Thank you
